website/azurerm.erb

```diff
@@ -71,6 +71,10 @@
                     <a href="/docs/providers/azurerm/d/api_management_product.html">azurerm_api_management_product</a>
                 </li>
 
+                <li<%= sidebar_current("docs-azurerm-datasource-api-management-property-x") %>>
```

Comment:
I only see a resource .go and no datasource?

Reply:
from what I can see, I don't think this Resource warrants a Data Source at this time - so I think this is fine 👍

Reply:
You're right here @katbyte, moved this link to the correct postition